lib/src/config/mod.rs

```diff
@@ -33,7 +33,7 @@
 //!   * **address**: _[string]_ an IP address or host the application will
 //!     listen on
 //!     * examples: `"localhost"`, `"0.0.0.0"`, `"1.2.3.4"`
-//!   * **port**: _[integer]_ a port number to listen on
+//!   * **port**: _[u16]_ a port number to listen on
```

Comment:
This should also say `integer` as it's referring to the TOML value, which doesn't know about `u16`. The description should say `a 16-bit unsigned integer` somewhere. Or maybe nothing. Probably better to just leave it alone.